x-pack/packages/kbn-securitysolution-data-table/README.md

```diff
@@ -0,0 +1,33 @@
+# Security Solutions's Data Table
+
+## Motivation
+
+The idea behind this package is to have a reusable data table component, embedding the features
+available to alerts table in security solution plugin.
+
+## How to use this
+
+Consult the following file to get the idea of what is necessary to reuse the component
+
+`x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx`
+
+The following file:
+`kibana/x-pack/plugins/security_solution/public/common/store/reducer.ts`
```

Comment:
Probably we should not reference fields in a static way here, they may change. It would be better to showcase an example explicitly, mentioning all context providers this package needs in order to work

Reply:
Fully fledged examples will follow, probably when integrating this with screens other than alerts.